src/js/Lotto.js

```diff
@@ -0,0 +1,47 @@
+import { WARNING_WHEN_NOT_IN_1000_UNITS, UNIT_PRICE } from "./utils/consts.js";
+
+class Lotto {
+  #price
+  constructor(price) {
+    this.#price = price
+  }
+
+  isPriceValid() {
+    if (this.#price < UNIT_PRICE || this.#price >= (UNIT_PRICE * 1000)) return false
+
+    if ((this.#price % UNIT_PRICE) !== 0) return this.valueInitAndAlertNotUnitOfthousand()
+
+    return true
+  }
+  
+  valueInitAndAlertNotUnitOfthousand() {
+    alert(WARNING_WHEN_NOT_IN_1000_UNITS)
+    return false;
+  }
+
+  get lottoTicketAmount() {
+    return this.isPriceValid() ? (this.#price / UNIT_PRICE) : null
+  }
+
+  static createRandomNumberFromOneToFortyFive(count) {
+    const lottoRandomNumbers = []
+    // 1 ~ 45 숫자로 구성된 배열 만들기
+    // 랜덤으로 셔플 하는 함수 가져오기
+    // 6개로 잘라서 가져옴
+    for (let i = 0; i < count; i++) {
+      while (true) {
+        const randomNumbers = Array.from({ length: 6 }, () => Math.floor(Math.random() * 45) + 1)
```

Comment:
1을 더해주고 있으므로 44가 맞지 않을까요?

> ( Math.random() * ( 최대값 - 최소값 )  ) + 최소값


```suggestion
        const randomNumbers = Array.from({ length: 6 }, () => Math.floor(Math.random() * 44) + 1)
```

Reply:
Math.random() 은 0 <= x < 1 의 범위를 가집니다.
다양한 방법이 있을 수 있을 텐데 저는
```javascript
(Math.random() * 45) // 0 <= x < 45
Math.floor(Math.random() * 45 + 1) // 1 <= x < 46
```
즉 최대 나올 수 있는 수는 45.9 고 버림을 하면 45라서 저렇게 작성했습니다!
저기서 44를 곱한다면 1 <= x < 45 가 될 것 같습니다.

Reply:
> Math.random() 은 0 <= x < 1 의 범위를 가집니다. 다양한 방법이 있을 수 있을 텐데 저는
> 
> ```js
> (Math.random() * 45) // 0 <= x < 45
> Math.floor(Math.random() * 45 + 1) // 1 <= x < 46
> ```
> 
> 즉 최대 나올 수 있는 수는 45.9 고 버림을 하면 45라서 저렇게 작성했습니다! 저기서 44를 곱한다면 1 <= x < 45 가 될 것 같습니다.

헉 ㅎㅎ 저는 Math.random() 이 0<= x <= 1 의 범위인줄 알았네요.. 😂 부끄럽습니다